shopsafe-frontend/src/environments/environment.ts

```diff
@@ -3,7 +3,10 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+
+  // URL of development API
+  apiUrl: 'http://localhost:8080'
```

Comment:
Yeah, does this work when we deploy?

Reply:
There's a different environment file for production versus development, so we will prob have to map the URL differently i in `environment.prod.ts` (opposed to `environment.ts`. This is a good point though, I'll look into it.